HCIDOCS-651: Oracle Cloud C3 (#88897)
openshift/openshift-docs: modules/c3-assisted-installer-completing-installation-manifests.adoc

// Module included in the following assemblies:
//
// * installing/installing_oci/installing-c3-assisted-installer.adoc

:_mod-docs-content-type: PROCEDURE
[id="c3-ai-completing-installation-manifests_{context}"]
= Adding custom manifests

> Create, modify and upload the four mandatory custom manifests provided by Oracle:

Reviewer: ```suggestion
Create, modify, and upload the four mandatory custom manifests provided by Oracle.
```

> ** In the `C3/custom_manifests_C3/manifests` folder:

Reviewer: ```suggestion
** In the `C3/custom_manifests_C3/manifests` folder, the following manifests are mandatory:
```

*** `oci-ccm.yml`
*** `oci-csi.yml`

> ** In the `C3/custom_manifests_C3/openshift` folder:

Reviewer: ```suggestion
** In the `C3/custom_manifests_C3/openshift` folder, the following manifests are mandatory:
```

*** `machineconfig-ccm.yml`
*** `machineconfig-csi.yml` 

.Prerequisites

> * Prepare the custom manifests. For details, see step 8 and onwards of link:https://www.oracle.com/a/otn/docs/compute_cloud_at_customer_assisted_installer.pdf?source=:em:nl:mt::::PCATP[Install the Cluster using the RH Assisted Installer UI].

Reviewer: ```suggestion
* Prepare the custom manifests. For details, see step 8 - 14 of the "Installing the Cluster using the RH Assisted Installer UI" section in the  link:https://www.oracle.com/a/otn/docs/compute_cloud_at_customer_assisted_installer.pdf?source=:em:nl:mt::::PCATP[Oracle documentation].
```

.Procedure

. Navigate to the *Custom manifests* page.

. Upload and save the `oci-ccm.yml` and `oci-csi.yml` files:

> .. In the *Folder* field, select `manifests`.

Reviewer: ```suggestion
.. In the *Folder* field, select *manifests*.
```

.. In the *File name* field, enter `oci-ccm.yml`.

.. In the *Content* section, click *Browse*. 

> .. Select the `oci-ccm.yml` manifest from the `C3/custom_ manifest_C3/manifests` folder.

Reviewer: ```suggestion
.. Select the *oci-ccm.yml* manifest from the `C3/custom_ manifest_C3/manifests` folder.
```

> .. Click *Add another manifest* and repeat the above steps for the `oci-csi.yml` file.

Reviewer: ```suggestion
.. Click *Add another manifest* and repeat the previous steps for the `oci-csi.yml` file.
```
For accessibility purposes, avoid directional language ("above", "below").

. Upload and save the `machineconfig-ccm.yml` and `machineconfig-csi.yml` files:

.. Click *Add another manifest*.

> .. In the *Folder* field, select `openshift`.

Reviewer: ```suggestion
.. In the *Folder* field, select *openshift*.
```

.. In the *File name* field, enter `machineconfig-ccm.yml`.

.. In the *Content* section, click *Browse*. 

> .. Select the `machineconfig-ccm.yml` file from the `C3/custom_ manifest_C3/openshift` folder.

Reviewer: ```suggestion
.. Select the *machineconfig-ccm.yml* file from the `C3/custom_ manifest_C3/openshift` folder.
```

> .. Click *Add another manifest* and repeat the above steps for the `machineconfig-csi.yml` file.

Reviewer: ```suggestion
.. Click *Add another manifest* and repeat the previous steps for the `machineconfig-csi.yml` file.
```

. Click *Next* to save the custom manifests.

> . From the *Review and create* page, click *Install cluster* to create your {product-title} cluster. This takes approximately thirty minutes.

Reviewer: ```suggestion
. From the *Review and create* page, click *Install cluster* to create your {product-title} cluster. This process takes approximately thirty minutes.
```

// Module included in the following assemblies:
//
// * installing/installing_oci/installing-c3-assisted-installer.adoc

:_mod-docs-content-type: PROCEDURE
[id="c3-ai-completing-installation-manifests_{context}"]
= Adding custom manifests

Create, modify and upload the four mandatory custom manifests provided by Oracle:

** In the `C3/custom_manifests_C3/`manifests` folder:

*** `oci-ccm.yml`
*** `oci-csi.yml`

** In the C3/custom_manifests_C3/`openshift` folder:

*** `machineconfig-ccm.yml`
*** `machineconfig-csi.yml` 

.Prerequisites

* Prepare the custom manifests. For details, see step 8 and onwards of link:https://www.oracle.com/a/otn/docs/compute_cloud_at_customer_assisted_installer.pdf?source=:em:nl:mt::::PCATP[Install the Cluster using the RH Assisted Installer UI].

.Procedure

> *[I am not sure this is correct. Do they create the files and paste the content in as in OCI, or just upload the files?]*

Author: Please note question here.
